demisto_sdk/commands/lint/tests/test_linter/gather_facts_test.py

```diff
@@ -38,6 +38,33 @@ def test_not_valid_yaml(self, demisto_content: Callable, create_integration: Cal
                                docker_engine=False)
         assert runner._gather_facts(modules={})
 
+    def test_checks_common_server_python(self, repo, demisto_content: Callable):
+        """
+
+        Given
+        - Repo with CommonServerPython script
+
+        When
+        - Running lint on a repo with a change to CommonServerPython
+
+        Then
+        - Validate that the CommonServerPython is in the file list to check
+        """
+        from demisto_sdk.commands.lint import linter
+        from wcmatch.pathlib import Path
```

Comment:
is there a reason this is imported in every test?

Reply:
That's how it worked before. I can move it up and change this file.

Reply:
lets move it, it doesnt make sense